docs/pages/freshness_check.md

```diff
@@ -0,0 +1,19 @@
+# Freshness check
+
+### Verify the freshness of the data
+
+```yaml
+dataset: postgres_adventureworks/adventureworks/advw/dim_employee
+
+checks:
+  - freshness:
```

Comment:
so freshness only works on dataset level, not supported to set it on the column ?

Reply:
Indeed.  I would  think it's best to have it only on dataset level.  If we ever want special Soda Cloud editor support for the single or first freshness check.  Seems better if that is on the predictable dataset level instead of searching for it in the columns.  Also freshness may later involve multiple columns.  That could also get more consistent with dataset level checks.